src/h5web/visualizations/nexus/hooks.ts

```diff
@@ -90,11 +85,8 @@ export function useNxData(group: HDF5Group, metadata: HDF5Metadata): NxData {
       dims: signalDataset.shape.dims,
       scaleType: signal_scale_type,
     },
-    errors,
-    title:
-      datasetValues && typeof datasetValues.title === 'string'
-        ? datasetValues.title
-        : undefined,
+    errors: errorsValue,
+    title: typeof values.title === 'string' ? values.title : undefined,
```

Comment:
I don't see how this is still working as the keys from `values` are now `HDF5Id`s.

Shouldn't you use the same trick as for `errors`: get the `title` link to get its `id` ?

Reply:
I see now why it isn't caught by the test on mock data:
- We use the name of the dataset as id in mock data
- The name of the title dataset is ... `title` sso that the id stays `title` :rofl: